CMakeLists.txt

```diff
@@ -179,7 +179,6 @@ if (${CMAKE_MAJOR_VERSION}.${CMAKE_MINOR_VERSION} GREATER 3.5) # Need 3.6 or abo
       ",-google-readability-todo"
       ",-google-runtime-int"
       ",-google-runtime-references"
-      ",-misc-misplaced-widening-cast"
```

Comment:
Why disable the check?

Reply:
It disables the "non-check" :)
It re-enables it 👍

Reply:
All checks are enabled, but the listed ones.